CWE/events/East Turkestan Additions.txt

```diff
@@ -0,0 +1,1404 @@
+			militancy = -3
+			consciousness = -3
+		}
+		ai_chance = { 
+			factor = 0.2
+		}
+	}
+}
+
+#Removal of Zhizhong
+country_event = {
+	id = 8023804
+	title = "Retirement of Zhizhong"
+	desc = EVTDESC8023804
+	picture = "xij_sabri"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = XIJ
+		year = 1947
+		NOT = { year = 1950 }
+
+		is_vassal = no
+		war = no
+
+		ruling_party_ideology = socialist
+
+		NOT = { XIJ = { war_with = ETS } } #Ili Rebellion
+	}
+
+	mean_time_to_happen = { months = 4 }
+
+	option = {
+		name = "Masud Sabri will ensure loyalty."
+		ruling_party_ideology = conservative
+		any_pop = {
+			limit = { OR = { culture = chinese culture = hui }
+			ideology = { factor = 0.10 value = conservative }
+			militancy = 3
+			consciousness = 3
+		}
+		ai_chance = { 
+			factor = 0.8
+		}
+	}
+
+	option = {
+		name = "Bai Chongxi will lead Xinjiang province as a model for the KMT."
+		add_country_modifier = { name = political_strife duration = 395 }
+		ruling_party_ideology = nationalist
+		any_pop = {
+			limit = { OR = { culture = chinese culture = hui }
+			ideology = { factor = 0.10 value = nationalist }
+			militancy = 3
+			consciousness = 3
+		}
+		ai_chance = { 
+			factor = 0.2
+		}
+	}
+}
+
+#Dismantling the reforms
+country_event = {
+	id = 8023805
+	title = "Dismantling the Reforms"
+	desc = EVTDESC8023805
+	picture = "xij_kmt"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = XIJ
+		year = 1947
+		NOT = { year = 1950 }
+
+		is_vassal = no
+		war = no
+
+		OR = { ruling_party_ideology = nationalist ruling_party_ideology = conservative }
+		NOT = { XIJ = { war_with = ETS } } #Ili Rebellion
+
+	}
+
+	mean_time_to_happen = { months = 3 }
+
+	option = {
+		name = "It is time to bring Xinjiang in line with the rest of China."
+		remove_accepted_culture = uigher
+
+		any_pop = {
+			limit = { OR = { culture = chinese culture = hui }
+			ideology = { factor = 0.10 value = nationalist }
+			militancy = 3
+			consciousness = 3
+		}
+		ai_chance = { 
+			factor = 0.7
+		}
+	}
+
+	option = {
+		name = "We will preserve what we have built."
+		add_country_modifier = { name = political_strife duration = 395 }
+		any_pop = {
+			limit = { OR = { culture = chinese culture = hui }
+			ideology = { factor = 0.10 value = socialist }
+			militancy = -3
+			consciousness = -3
+		}
+		ai_chance = { 
+			factor = 0.2
+		}
+	}
+}
+
+#Soviet Propaganda
+country_event = {
+	id = 8023806
+	title = "Continued Efforts in East Turkestan"
+	desc = EVTDESC8023806
+	picture = "nwo2_east_turkestan"
+
+	fire_only_once = yes
+
+	trigger = {
+		tag = RUS
+		year = 1947
+		NOT = { year = 1950 }
+
+		war = no
+
+		OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 }
+
+		ETS = { 
+			exists = no
+		}
+		XIJ = { 
+			exists = yes
+			government = populist_dictatorship
+			OR = { NOT = { ruling_party_ideology = socialist ruling_party_ideology = communist ruling_party_ideology = communist_social } }

```

Comment:
NOT = { OR = { ruling_party_ideology = socialist ruling_party_ideology = communist ruling_party_ideology = communist_social } }